tests/BehatFeatures/web/translation/description_custom_translation.feature

```diff
@@ -9,87 +9,62 @@ Feature: Projects should have descriptions where a custom translation can be def
       | id | name      | owned by | description    |
       | 1  | project 1 | Catrobat | my description |
 
-  Scenario: Custom translation editor should be visible
-    Given I log in as "Catrobat"
-    And I go to "/app/project/1"
-    And I wait for the page to be loaded
-    When I click "#edit-description-button"
-    And I wait for AJAX to finish
-    Then the element "#description" should not be visible
-    But the element "#edit-text" should be visible
-    And the element "#edit-text-ui" should be visible
-    And the element "#edit-language-selector" should be visible
-    And the element "#edit-submit-button" should be visible
-
   Scenario: Adding a custom description translation
```

Comment:
I think this test case can also be deleted

Reply:
Deleted the test case